better-text-drawings.js

```diff
@@ -52,7 +52,12 @@ Hooks.on('setup', () => {
 
 		// Make textInput a text area
 		let textInput = html.find('input[name="text"]');
-		let textArea = $('<textarea name="text" data-dtype="String"></textarea>');
+		let parentWindow = textInput.parent().parent().parent().parent().parent();
```

Comment:
Not a fan of this mass `parent()` chain. You probably should be using jQuery methods to search up for the application window.

Reply:
I'll change this to libWrapper after merging.